build/geniuspaste.m4

```diff
@@ -3,7 +3,8 @@ AC_DEFUN([GP_CHECK_GENIUSPASTE],
     GP_ARG_DISABLE([GeniusPaste], [auto])
 
     GP_CHECK_PLUGIN_DEPS([GeniusPaste], GENIUSPASTE,
-                         [libsoup-2.4 >= 2.4.0])
+                         [gtk+-3.0 >= 3.16
```

Comment:
We also need to adjust the CI workflow config to install libsoup3. Currently the plugin will not be built in the CI due to the missing dependency.

Reply:
Indeed, trying it now -- but that also requires bumping the Ubuntu version because 20.04 doesn't have it.

Reply:
@eht16 Seems to work, but that hit the cppcheck issue again, do you know anything about that?

Reply:
No. It breaks also the nightly builds and so I merged #1310 today to unhide this problem.
Unfortunately, now it is not hidden any longer.